tests/circuit_graph/test_circuit_graph_hash.py

```diff
@@ -295,6 +192,7 @@ def circuit2(x, y):
         "x,y",
         zip(np.linspace(-2 * np.pi, 0, 3), np.linspace(-2 * np.pi, 0, 3)),
     )
+    @pytest.mark.xfail(reason="This test will not work in tape mode")
```

Comment:
Better to just remove or update this test?

Reply:
deleted :)